### PR TITLE
feat(init): migrate to provd based telemetry service

### PR DESCRIPTION
**Pull request by d-loose**
Removes the sysmetrics package and corresponding telemetry service in favor of a provd based service.

~~e2e test and go code check are expected to fail as the provd implementation is still missing.~~

Ref #363
UDENG-2204

### CODE REVIEW

**Comment by d-loose** [comment]
Errors returned by provd are ignored for now. I'd merge the UI update from #667 first and think about error handling afterwards.

File: ci/snap/init/snapcraft.yaml
Comment (d-loose) [inline ci/snap/init/snapcraft.yaml:1]: @matthew-hagemann I suppose we can remove this library from the snap since we're now using the native Go API

**Review by spydon — APPROVED** [review]
Nice, lgtm!

File: ci/snap/init/snapcraft.yaml
Comment (d-loose) [reply ci/snap/init/snapcraft.yaml:1]: .. and since the entire code for this moved to the deb package anyways.. :D

**Pull request merged**